elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/storage/ElasticsearchIndex.java

```diff
@@ -87,6 +92,20 @@ public PhysicalPlan implement(LogicalPlan plan) {
      * index scan.
      */
     return plan.accept(new DefaultImplementor<ElasticsearchIndexScan>() {
+          @Override
+          public PhysicalPlan visitFilter(LogicalFilter node, ElasticsearchIndexScan context) {
```

Comment:
What happen if there are multiple filter, e.g. source=index | filter a = 1 | filter b = 1

Reply:
Good point. Will double check and add IT to make sure this can be handled properly. Thanks!

Reply:
Added logic to handle multiple filter operator pushdown and UT/IT to test. We may need to improve our planner soon since current index scan operator assumes all push down is around original data source. I guess if we do `where | stats | where`, the result would be wrong.